Library/Formula/libssh.rb

```diff
@@ -14,6 +14,7 @@ class Libssh < Formula
 
   depends_on "cmake" => :build
   depends_on "openssl"
+  depends_on "krb5" => :build unless OS.mac?
```

Comment:
No idea. But `gssapi.h` (and, therefore, `krb5`) is definitely necessary to build `libssh`.